docs/paper/paper.md

```diff
@@ -21,41 +21,40 @@ link-bibliography: false
+The core features of PlasticParcels are: 1) a user-friendly python notebook layer on top of `Parcels` that provides a streamlined workflow for performing plastic dispersal simulations, 2) custom `Parcels` kernels designed to simulate the fine-scale physical processes that influence the transport of nano- and microplastic particulates, and 3) global particle initialisation maps which represent the best estimate locations of plastic pollution emissions from coastal sources, river sources, open ocean fishing-related activity emission sources, and a current best estimate of buoyant plastic concentrations. We visualise these initialisation maps in Fig. \ref{fig:initialisation_maps}.
 
-In addition, due to the flexibility of the package, users may use the functions and modular design of `PlasticParcels` to enhance their existing `Parcels` simulations and workflow. For example, users can use the initialisation maps, associated `ParticleSet` creation methods, and/or the custom physics kernels with their own `Parcels` simulations. Post-processing and analysis of the generated trajectory datasets is purposefully left to the user, however some tutorials are provided in the [`PlasticParcels` github repository](\url{https://github.com/OceanParcels/PlasticParcels}), along with the tutorials on the [`Parcels` github repository](\url{https://github.com/OceanParcels/parcels}). Below we provide an example of how `PlasticParcels` may be used, utilising a developed `ipynb` tutorial notebook.
+![Particle initialisations maps based on a) coastal mismanaged plastic waste emissions [@Jambeck2015], b) riverine mismanaged plastic waste emissions [@Meijer2021], c) fishing activity related plastic emissions [@Kroodsma2018], and d) a current global surface concentration estimate [@Kaandorp2023].\label{fig:initialisation_maps}](initialisation_maps.png){width=80%}
 
+In addition, due to the flexibility of the package, users may use the functions and modular design of `PlasticParcels` to enhance their existing `Parcels` simulations and workflow. For example, users can use the initialisation maps, associated `ParticleSet` creation methods, and/or the custom physics kernels with their own `Parcels` simulations. Post-processing and analysis of the generated trajectory datasets is purposefully left to the user, however some tutorials are provided in the [`PlasticParcels` github repository](\url{https://github.com/OceanParcels/PlasticParcels/docs/examples}), along with the tutorials in the [`Parcels` documentation](\url{https://docs.oceanparcels.org/en/latest/documentation.html}). Below we provide an example use case of `PlasticParcels`.
```

Comment:
Refer to the documentation website instead of the GitHub repo?

Reply:
Updated to documentation page instead of github repo